tools/container/runtime/containerd/containerd.go

```diff
@@ -164,3 +167,19 @@ func (o *Options) runtimeConfigOverride() (map[string]interface{}, error) {
 
 	return runtimeOptions, nil
 }
+
+func GetLowlevelRuntimePaths(hostRoot string) ([]string, error) {
+	cfg, err := containerd.New(
+		containerd.WithConfigSource(containerd.CommandLineSource(hostRoot)),
+	)
```

Comment:
Thinking about this again, we probably want to include the same options (with the exception of the config file) here: 
```suggestion
func GetLowlevelRuntimePaths(o *container.Options, co *Options) ([]string, error) {
	cfg, err := containerd.New(
		containerd.WithConfigSource(containerd.CommandLineSource(o.HostRootMount)),
		containerd.WithRuntimeType(co.runtimeType),
		containerd.WithUseLegacyConfig(co.useLegacyConfig),
	)
```